yt/visualization/plot_container.py

```diff
@@ -892,10 +893,11 @@ def set_background_color(self, field, color=None):
         if color is None:
             cmap = self._colormap_config[field]
             if isinstance(cmap, str):
-                try:
-                    cmap = yt_colormaps[cmap]
-                except KeyError:
-                    cmap = getattr(matplotlib.cm, cmap)
+                cmap = (
+                    yt_colormaps[cmap]
+                    if cmap in yt_colormaps
+                    else mpl_cmap_registry[cmap]
+                )
```

Comment:
while this should be equivalent to the previous version, it should be noted that

```python
cmap = mpl_cmap_registry[cmap]
```
does the same thing